package.json

```diff
@@ -4,11 +4,13 @@
   "repository": "https://github.com/dimagi/commcare-hq",
   "description": "CommcareHQ Javascript tests",
   "dependencies": {
+    "biginteger": "^1.0.3",
```

Comment:
I don't think you need this since it's required by js-xpath and you don't use it explicitly here?